ValveKeyValue/ValveKeyValue/KVSerializerOptions.cs

```diff
@@ -26,6 +23,29 @@ public string[] Conditions
         /// </summary>
         public IIncludedFileLoader FileLoader { get; set; }
 
+        public KVSerializerOptions()
+        {
+            Conditions ??= new List<string>();
```

Comment:
wait why would this ever be null in the constructor?

Reply:
are init properties executed after constructor? like so: `new KVSerializerOptions { Conditions = abc }`

Reply:
Property assignments like that always run after the constructor.